doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc

```diff
@@ -3,191 +3,54 @@
 
 This section highlights new features in {PRODUCT} {PRODUCT_VERSION}.
 
-ifdef::DM[]
+== Ability to only enable decision management components and functionality
 
-== {PRODUCT} {ENTERPRISE_VERSION} Kogito decision services support
+You can now configure the deployment to only enable decision management related components and functionality so that jBPM and case management features are disabled.
 
-=== IBM Power support in {OPENSHIFT} 4.10
+== Ability to use a custom registry and specific images
 
-Starting with {PRODUCT} {ENTERPRISE_VERSION}, IBM Power supports {OPENSHIFT} 4.10 using Open JDK 11. For a complete list of supported configurations, see https://access.redhat.com/articles/3354301[Red Hat Decision Manager 7 Supported Configurations].
+You can now configure the custom registry to use specific images during the configuration of the environment.
 
-endif::DM[]
+== Ability to output error logs
```

Comment:
I don't think this title matches the content well, I would suggest something like: Eager initialization of Kafka emitters for jBPM

Reply:
This should also not be in the RHDM release notes

Reply:
Noted